examples/agent/main.tf

```diff
@@ -21,8 +21,8 @@ resource "aws_vpc_endpoint" "s3" {
 module "apollo" {
   source = "../../"
 
-  cloud_account_id = "682816785079"
-  image            = "404798114945.dkr.ecr.*.amazonaws.com/mcd-pre-release-agent:latest"
+  cloud_account_id = "590183797493"
+  image            = "590183797493.dkr.ecr.*.amazonaws.com/mcd-agent:latest"
```

Comment:
Don't forget to update the example [README](https://github.com/monte-carlo-data/terraform-aws-mcd-agent/blob/af2326664d52e7552316fffdad5a2e3c71bec640/examples/agent/README.md?plain=1#L6) too. Thanks!

Reply:
And this [README](https://github.com/monte-carlo-data/terraform-aws-mcd-agent?tab=readme-ov-file#inputs) too. Still pointing to the old account. Thanks!